wiki/Game_mode/pt-br.md

```diff
@@ -3,28 +3,57 @@ outdated_translation: true
 outdated_since: d5b45241bf64c7abeadfa160238f428bbcaa4d2a
 ---
```

Comment:
if this translation is translated on latest en.md, then this must be deleted
erase line 1 to 5

Reply:
ok!